### DISCUSSION TITLE
Is `ignorePaths` a `mergeable` option?

### DISCUSSION THREAD

**Discussion by nils-a** [renovatebot/renovate#14941] [Request Help] [answered]
Some options in the docs state `mergeable` being either `true` or `false`. However, some (like [`ignorePaths`](https://docs.renovatebot.com/configuration-options/#ignorepaths)) do not state anything about being `mergeable` or not.

My question is this:
given a configuration like:
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "npm": {
    "ignorePaths": [
      "docs/**"
    ]
  },
  "ignorePaths": [
    "resources/scripts/**"
  ]
}
```

The docs state that the default for `ignorePaths` is `["**/node_modules/**", "**/bower_components/**"]` (probably augmented a bit by inheriting from `config:base`)

What would the `ignorePaths` value be, for the `npm` module? 
Ideally, I would expect  `["**/node_modules/**", "**/bower_components/**",  "resources/scripts/**", "docs/**"]`

**Comment by rarkins** [comment, marked as answer]
It's false, so that you can "unignore" any defaults, such as node_modules

**Reply by nils-a** [reply]
Thanks @rarkins !

**Reply by heapifyman** [reply]
> It's false, so that you can "unignore" any defaults, such as node_modules

@rarkins Is it possible to "unignore" sub-directories or files of an ignored path? Like in `.gitignore` files?

Say I want to ignore the default `**/test/**` but **un**ignore one particular file within the `test` directory, like `**/test/foo/bar/do-not-ignore-this.file`. How would I go about doing that?

I see that there is `includePaths` option but it seems that one can use either `ignorePaths` or `includePaths` but not both?

**Comment by HonkingGoose** [comment]
I'm trying to fix our docs with this PR:

- https://github.com/renovatebot/renovate/pull/14946.
